repository/kafka/docs/latest/limitations.md

```diff
@@ -0,0 +1,21 @@
+
+
+# Limitations
+
+
+
```

Comment:
Can we reduce all the 3-line-breaks to 1? The rest of your docs follow this pattern (and it's easier to read in code-form)

Reply:
done!